easybuild/easyconfigs/s/SignalP/SignalP-6.0h-foss-2024a-fast.eb

```diff
@@ -0,0 +1,50 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonPackage'
+
+name = 'SignalP'
+version = '6.0h'
+_suffix = 'fast'
+versionsuffix = '-' + _suffix
+
+homepage = 'https://services.healthtech.dtu.dk/software.php'
+description = """SignalP predicts the presence  and  location of signal peptide  cleavage sites
+in  amino acid  sequences  from  different organisms"""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+download_instructions = """
+SignalP requires registration and acceptance of licence terms (academic use only).
+ [1] go to: https://services.healthtech.dtu.dk/service.php?SignalP-6.0
+ [2] navigate to the "Downloads" tab
+ [3] select "%s" type under version "%%(version)s"
+ [4] complete the form; you should receive a download link via email
+""" % _suffix
+
+sources = ['%%(namelower)s-%%(version)s.%s.tar.gz' % _suffix]
+checksums = ['4afe3c004e23a1d6518cefb23448b4d5ede5f1c5b276db541f839f0615822125']
+
+unpack_options = '--strip-components=1'
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),    # numpy > 1.19.2
+    ('tqdm', '4.66.5'),             # tqdm > 4.46.1
+    ('PyTorch', '2.6.0'),          # torch > 1.7.0
+    ('matplotlib', '3.9.2'),        # matplotlib > 3.3.2
+]
+
+_bin = '%%(namelower)s%s' % version[0]
+sanity_check_paths = {
+    'files': ['bin/%s' % _bin],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
+}
+
+preinstallopts = "sed -i 's/torch.*/torch/g' requirements.txt && "
```

Comment:
sorry for the nitpick, but can you move `preinstallopts` back to above the sanity_check_paths?